packages/e2e/test/blockfrost/getAsset.test.ts

```diff
@@ -1,6 +1,7 @@
 import * as envalid from 'envalid';
 import { Cardano } from '@cardano-sdk/core';
 import { assetProviderFactory } from '../../src/factories';
+import { dummyLogger } from 'ts-log';
```

Comment:
```suggestion
import { TestLogger } from 'util-dev';
```

Reply:
I force pushed the changes because they are contained and simple

Reply:
We agreed offline that webextension logs are ok to go to the browser console because that is where they will execute